Podfile

```diff
@@ -121,6 +121,13 @@ abstract_target 'defaults' do
 end
 
 post_install do |installer|
+  installer.pod_targets.each do |target|
+    useAsSystemFramework = %w[ React ]
+    next unless useAsSystemFramework.include? target.name
+    # headers from system frameworks don't generate warnings
+    system("sed", "-i.bak", "/^framework module / s/{/[system] {/", target.module_map_path.to_path)
```

Comment:
Making React a system framework seems like it may have additional side effects. It sounds scary. Do `inhibit_all_warnings!` and `:inhibit_warnings => true` not work?
https://stackoverflow.com/questions/31644959/how-can-i-silence-warnings-from-all-pods-except-local-pods

Reply:
They don't seem to work with frameworks that have modules in them.